source/index.html.md

```diff
@@ -436,6 +436,421 @@ Definition of the object attributes
+
+| Attribute                               |  Description                                                      |
+| --------------------------------------- | ----------------------------------------------------------------- |
+| id                 | Unique identifier of the Organization within the Shipamax system |
+| externalId                               | Unique identifier of the Organization within your own system           |
+| carrier                       | Flag for denoting this Organization is a carrier   |
+| consignee                       | Flag for denoting this Organization is a consignee   |
+| creditor                       | Flag for denoting this Organization is a creditor   |
+| forwarder                       | Flag for denoting this Organization is a forwarder   |
+| debtor                       | Flag for denoting this Organization is a debtor   |
+| shipper                       | Flag for denoting this Organization is a shipper   |
+| active                       | Flag denoting wether this Organization is active or not   |
+| updated | The timestamp of when the Organization was last updated |
+
+### POST
+Send a new Organization via `POST` request to `https://public.shipamax-api.com/api/v2/Organizations`
+
+> The POST Organizations request requires a body JSON structured like this:
+```json
+{
+    "externalId": string,
+    "carrier": boolean,
+    "consignee": boolean,
+    "creditor": boolean,
+    "forwarder": boolean,
+    "debtor": boolean,
+    "shipper": boolean,
+    "active": boolean
+}
+```
+
+> Example of body to be POSTED:
+
+```json
+{
+    "externalId": "FF",
+    "carrier": false,
+    "consignee": true,
+    "creditor": true,
+    "forwarder": false,
+    "debtor": false,
+    "shipper": false,
+    "active": false
+}
+```
+
+>The POST endpoint will respond with JSON like this:
+```json
+{
+  "id": 9,
+  "externalId": "FF",
+  "carrier": false,
+  "consignee": true,
+  "creditor": true,
+  "forwarder": false,
+  "debtor": false,
+  "shipper": false,
+  "active": false,
+  "updated": "2020-01-01T00:00:00.000Z"
+}
+```
+
+**NOTE: The ID returned by the endpoint is needed to be able to create and relate addresses and names to the Organization**
+### GET
+Retrieve an Organization via `GET` request to `https://public.shipamax-api.com/api/v2/Organizations/${id}`
+
+>The GET endpoint responds with JSON like this:
+```json
+{
+  "id": 2,
+  "externalId": "XXXYYY",
+  "carrier": true,
+  "consignee": true,
+  "creditor": true,
+  "forwarder": false,
+  "debtor": false,
+  "shipper": true,
+  "active": true,
+  "updated": "2021-08-06T09:58:20.384Z"
+}
+```
+
+### GET (Using Query Filter)
+
+Retrieve Organizations via `GET` request to `https://public.shipamax-api.com/api/v2/Organizations` and sending in a JSON body to return all Organizations matching the given filter body, or none for all Organizations.
+
+> The GET Organizations query filter body requires JSON structured like this:
+```json
+{
```

Comment:
Can we please add an example of a using the GET with filter

Reply:
This part is the example itself?